scipy/linalg/tests/test_decomp.py

```diff
@@ -43,6 +43,7 @@
 
 def _random_hermitian_matrix(n, posdef=False, dtype=float):
     "Generate random sym/hermitian array of the given size n"
+    # FIXME non-deterministic rng
```

Comment:
I just wanted to double check that this is a future fix me and not one that was intended for this pr

Reply:
yes, it's left for the future as fixing it would involve major refactoring. It's non-deterministic with the GIL too, so not terribly concerned.